server/graphql/v2/query/collection/AccountsQuery.ts

```diff
@@ -103,7 +104,12 @@ const AccountsQuery = {
       where['HostCollectiveId'] = hostsWithSupportedPaymentProviders.map(h => h.id);
 
       // Fetch & return results
-      const order = [[args.orderBy.field, args.orderBy.direction]];
+      const orderBy = args.orderBy || { field: 'CREATED_AT', direction: 'DESC' };
+      if (orderBy.field !== 'CREATED_AT') {
+        throw new Error(`Only CREATED_AT is supported for orderBy when using supportedPaymentMethodService`);
+      }
+
+      const order = [['createdAt', args.orderBy.direction || 'DESC']];
```

Comment:
Defaulting to `createdAt, DESC` for this one to match the behavior initially implemented for hacktoberfest with `CHRONOLOGICAL_ORDER_INPUT_DEFAULT_VALUE` https://github.com/opencollective/opencollective-api/pull/6558/files#diff-614137bfb13507896874ec467e5806f73d040385707a14d5d6188b88cf1c3b90R37